04-copy-directory/README.md

```diff
@@ -1,39 +1,39 @@
-## Копирование папки
+## Copying a Directory
 
-В файле **index.js** директории **04-copy-directory** реализуйте функцию **copyDir** которая копирует содержимое папки **files** в папку **files-copy**.
+In the `index.js` file of the `04-copy-directory directory`, implement the `copyDir` function, which copies the contents of the files folder to the `files-copy` folder.
```

Comment:
In the `index.js` file of the `04-copy-directory` directory, implement the `copyDir` function, which copies the contents of the files folder to the `files-copy` folder.